ui/src/app/pages/homepage/data-access-requirements.spec.tsx

```diff
@@ -37,17 +38,103 @@ describe('DataAccessRequirements', () => {
         profileStore.set({profile, load, reload, updateCache});
     });
 
-    it('should render all modules by default', () => {
-        const wrapper = component();
-        allModules.forEach(module => expect(findModule(wrapper, module).exists()).toBeTruthy());
+    it('should return all modules from getEnabledModules by default (all FFs enabled)', () => {
```

Comment:
most of these changes are to confirm to myself that the state/render logic is robust.  Scroll down for the relevant change.